contracts/VaultV3.vy

```diff
@@ -663,28 +649,31 @@ def _process_report(strategy: address) -> (uint256, uint256):
 
 @view
 @internal
-def _estimate_unlocked_profit() -> uint256:
+def _compute_unlocked_profit() -> uint256:
     """
-    If profit_distribution_rate is equal to zero, there is no profit to unlock, otherwise we estimate it
+    If profit_distribution_rate is equal to zero, there is no profit to unlock, otherwise we compute it
     """
-    if self.profit_distribution_rate != 0:
+    # Hack: to save gas
+    profit_distribution_rate: uint256 = self.profit_distribution_rate_
+    profit_last_update: uint256 = self.profit_last_update
```

Comment:
not needed as you only use it once after this call. (and if needed, it would be better to do this just within the following if-else

Reply:
you are right